xorriso/gen_input.sh

```
@@ -44,7 +44,7 @@ then
 fi
 
 pushd "${PWD}/../container" > /dev/null
-for file in $(find "${IMAGE_NAME}-${IMAGE_TAG}" -type f)
+for file in $(find "${IMAGE_NAME}" -type f)
```

Shellcheck (0.11.0)

[warning] For loops over find output are fragile. Use find -exec or a while read loop.

(SC2044)